tools/provision.sh

```diff
@@ -12,8 +12,11 @@ if [[ $ID == rhel || $ID == centos ]]; then
 fi
 
 sudo mkdir -p /etc/osbuild-composer
-sudo cp -a /usr/share/tests/osbuild-composer/composer/*.toml \
+sudo mkdir -p /etc/osbuild-worker
+sudo cp -a /usr/share/tests/osbuild-composer/composer/osbuild-composer.toml \
     /etc/osbuild-composer/
+sudo cp -a /usr/share/tests/osbuild-composer/composer/osbuild-worker.toml \
```

Comment:
Maybe we should have a separate `worker` subdirectory which is copied to `/etc/osbuild-worker`?

Reply:
Yeah, that probably makes sense. Both composer and worker configurations are currently stored in `test/data/composer` directory in git and installed to this directory.

Reply:
Right, that seems like a mistake. Should have been split all the way.

Reply:
This is fixed now.